api-docs/rst/dev-guide/concepts.rst

```diff
@@ -655,7 +768,7 @@ Autoscale supports a cooldown feature. A cooldown is a configured
 period of time that must pass between actions. Cooldowns only apply to
 webhook-based configurations. By configuring group cooldowns, you
 control how often a group can have a policy applied, which can help
-servers scaling up to complete the scale up before another policy is
+units scaling up to complete the scale up before another policy is
 executed. By configuring policy cooldowns, you control how often a
```

Comment:
you can control how often a policy can be applied to a group which can help prevent another policy from being applied while units are scaling up.